src/transformers/models/gpt_neo/modeling_gpt_neo.py

```diff
@@ -50,6 +63,36 @@
 _CHECKPOINT_FOR_DOC = "EleutherAI/gpt-neo-1.3B"
 
 
+# TODO: @younesbelkada move that in pytorch_utils and document it
+if is_torch_fx_available():
```

Comment:
I will take care of that in a follow up PR, this seems to be needed to make the torch_fx pass

Reply:
No no, let's wait until your PR is merged 😉

Reply:
Hello @younesbelkada, sorry for disturbing but did you manage to add this method to pytorch_utils?

I would then update the code to make it ready for another review.